.github/workflows/ci.yml

```diff
@@ -30,3 +30,13 @@ jobs:
         run: swift test
       - name: Compile Generated Code
         run: ./Scripts/test-generated.sh
+  linux:
+    name: Linux
+    runs-on: ubuntu-latest
+    container: swift:5.6
```

Comment:
Maybe I should run this on Swift 5.5 as well? Same as we are doing with the Xcode matrix?